adding a new page with ecub1 joint specs

divyashah opened the pull request:
This is regarding #184. 

I have gathered the information regarding all the specifications for the joints of ergoCub 1 and created a new page. I mainly looked into the CAD model and the configuration files from `robots-configuration`. It would be worth reviewing these information as there might have been some ongoing upgrades which might have been reported yet in the source files I have considered, especially w.r.t hands. 

cc @pattacini @maggia80

pattacini commented:
Thanks @divyashah 👍🏻 
I'll be reviewing the PR in the background.

In the meantime:
- Can you please replace all the occurrences of `ecub` into `ergoCub`, both in file names and text? We don't use the tag `ecub`, in fact.
- For checking the hands, would you be willing to coordinate with for example @Lawproto and/or @ale-git?

divyashah commented:
> I'll be reviewing the PR in the background.

Thanks @pattacini 

> * Can you please replace all the occurrences of `ecub` into `ergoCub`, both in file names and text? We don't use the tag `ecub`, in fact.

With the previous commit: https://github.com/icub-tech-iit/documentation/pull/297/commits/073334044d70fe59121c56c928acf0016c25fa5f, I updated `ecub` to `ergoCub`. 

> * For checking the hands, would you be willing to coordinate with for example @Lawproto and/or @ale-git?

I discussed with @Lawproto. 
There has been a modification to the RoM of `thumb_add` joint, I updated the hardware limits for the same in https://github.com/icub-tech-iit/documentation/pull/297/commits/b4abaa447f8536a6c93f530b7077ff6a3758cd40.

pattacini commented on docs/ergoCub1/ergoCub1-joints.md:
```diff
@@ -0,0 +1,332 @@
+| Range of Motion (HW) | -92/+92 (degrees)                                                             |
+| Range of Motion (SW) | -90/+90 (degrees)                                                             |
+
+> PS: all wrist joints are coupled into a co-axial spherical parallel mechanism
```
Please, insert a blank line here.

divyashah commented:
> More importantly, the order of the fingers is not ok. You followed the mechanical order, whereas the joints are published remapped as per https://github.com/robotology/robots-configuration/blob/master/ergoCubSN000/wrappers/motorControl/left_arm-mc_remapper.xml#L5.
> 
> Hence, could you please rearrange the doc?

Hi @pattacini,

Thanks for pointing it out. I've rearranged the order of the hand joints as you suggested via the commit https://github.com/icub-tech-iit/documentation/pull/297/commits/3f35659a1979f845409a61a572f0742cad033e3b.

> Probably, it's better to replace the `PS` with a mkdocs [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/). A `note` should be fine.

Also, I added the admonition notes via https://github.com/icub-tech-iit/documentation/pull/297/commits/6590b737ef38ff6862a8b0437cb27345673e57c3.

pattacini reviewed (approved):
Well done!
Thanks @divyashah!
Merging.
